.github/workflows/release.yml

```diff
@@ -0,0 +1,193 @@
+name: Release
+on:
+  release:
+    types:
+      - created
```

Comment:
For VS Code, I had `published` here:

https://github.com/erlang-ls/vscode/blob/main/.github/workflows/publish.yml#L4